packages/core/src/integrations/zoderrors.ts

```diff
@@ -5,33 +5,45 @@ import { truncate } from '../utils-hoist/string';
 
 interface ZodErrorsOptions {
   key?: string;
+  /**
+   * Limits the number of Zod errors inlined in each Sentry event.
+   *
+   * @default 10
+   */
   limit?: number;
+  /**
+   * Save full error info as an attachment in Sentry
+   *
+   * @default false
+   */
+  saveAttachments?: boolean;
```

Comment:
I'd like to make this more clear in terms of what exactly is being added to the attachment. Feel free to adjust the name again - naming is hard 😅 

```suggestion
  saveZodIssueAsAttachment?: boolean;
```

Reply:
Done!